build/template-run-unit-tests.yaml

```diff
@@ -3,18 +3,57 @@
 
 steps:
 
+- task: VSTest@2
+  displayName: 'Run integration tests (.NET 8)'
+  inputs:
+    testSelector: 'testAssemblies'
+    testAssemblyVer2: 'Tests\**\WebAppUiTests\**\net8.0\WebAppUiTests.dll'
+    searchFolder: '$(System.DefaultWorkingDirectory)'
+    rerunFailedTests: true
+    rerunMaxAttempts: '3'
```

Comment:
This is great!

Reply:
out of curiosity why is it great?

Reply:
Because it allows integration tests to rerun with 3 attempts if the test is flaky. Most of the time, the flaky tests we have in MSAL are worked around using this feature, especially for tests where it is challenging to find the root cause of the flakiness. These tests will show up in the "Passed on rerun" category in the build results. Having a test rerun itself and passing is better than the build failing entirely and needing to be rerun manually.

![image](https://github.com/AzureAD/microsoft-identity-web/assets/30090357/d06d1e1b-8e6d-43bd-893d-fd9327240be6)